chapter-9-continuous-deployment/.vagrant/machines/node1/virtualbox/vagrant_cwd

```diff
@@ -0,0 +1 @@
+/home/chin-pompey/Documents/tools-examples/learning-ci-jenkins-exercises-material/chapter-9-continuous-deployment
```

Comment:
It's an absolute path that won't work for other users...

Reply:
I have created a .gitignore file to exclude those files.